ingest-app/src/cmr/ingest/data/bulk_update.clj

```diff
@@ -115,34 +117,34 @@
         (j/db-do-prepared db statement [status status-message task-id]))
       (catch Exception e
         (errors/throw-service-error :invalid-data
-         [(str "Error creating updating bulk update task status "
-               (.getMessage e))]))))
+                                    [(str "Error creating updating bulk update task status "
+                                          (.getMessage e))]))))
 
   (update-bulk-update-collection-status
     [db task-id concept-id status status-message]
     (try
       (j/with-db-transaction
-       [conn db]
-       (let [statement (str "UPDATE bulk_update_coll_status "
-                            "SET status = ?, status_message = ?"
-                            "WHERE task_id = ? AND concept_id = ?")
-             status-message (util/trunc status-message 255)]
-         (j/db-do-prepared db statement [status status-message task-id concept-id])
-         (let [task-collections (su/query db
-                                 (su/build (su/select
-                                            [:concept-id :status]
-                                            (su/from "bulk_update_coll_status")
-                                            (su/where `(= :task-id ~task-id)))))
-               pending-collections (filter #(= "PENDING" (:status %)) task-collections)
-               failed-collections (filter #(= "FAILED" (:status %)) task-collections)]
-           (when-not (seq pending-collections)
-             (update-bulk-update-task-status db task-id "COMPLETE"
-               (generate-task-status-message
-                 (count failed-collections) (count task-collections)))))))
+        [conn db]
```

Comment:
This is one place the original indentation matched our style guide (all your other changes to indentation looked good). If there are no arguments on the function call line we indent the first argument a single space on the next line.

https://github.com/bbatsov/clojure-style-guide#one-space-indent